system-tests/lib/dep-installer/index.ts

```diff
@@ -99,7 +99,7 @@ function getRelativePathToProjectDir (projectDir: string) {
 async function restoreLockFileRelativePaths (opts: { projectDir: string, lockFilePath: string, relativePathToMonorepoRoot: string }) {
   const relativePathToProjectDir = getRelativePathToProjectDir(opts.projectDir)
   const lockFileContents = (await fs.readFile(opts.lockFilePath, 'utf8'))
-  .replaceAll(opts.relativePathToMonorepoRoot, relativePathToProjectDir)
+  .replaceAll(opts.relativePathToMonorepoRoot.replace(/\\+/g, '/'), relativePathToProjectDir.replace(/\\+/g, '/'))
```

Comment:
Why was this change necessary?

Reply:
This was neccessary to be able to run a project with a linked project in the yarn lockfile in windows. yarn always uses `/` to resolve paths, the path-seperators used by path however change depending on the os.

Reply:
Right, I see. We only run the system-tests on linux on CI normally, and most developers are on linux/mac, so I suppose that's why no-one had encountered this before.